src/app/screens/Home/index.tsx

```diff
@@ -94,6 +100,36 @@ function Home() {
     }
   }
 
+  async function onTabChangeHandler(index: number) {
+    if (index === 1) {
+      await loadInvoices();
+    }
+  }
+
+  async function loadInvoices() {
+    // incoives have already been loaded
+    if (invoiceTransactions) return invoiceTransactions;
```

Comment:
switching accounts does not reload the invoices.

we also don't do this with the outgoing ones because there we only show the ones in our DB (no matter from which account)